targets/arty/base.py

```diff
@@ -19,6 +19,7 @@
 class BaseSoC(SoCSDRAM):
     mem_map = {**SoCSDRAM.mem_map, **{
         'spiflash': 0x20000000,
+        'emulator_ram': 0xa0000000
```

Comment:
@mithro - I have no idea what the correct offset of `emulator_ram` should be, but withtout this entry in `mem_map` the platform failed on Travis.

Reply:
Should it be the same as what ever https://github.com/litex-hub/linux-on-litex-vexriscv uses?

Reply:
Up to this commit: https://github.com/litex-hub/linux-on-litex-vexriscv/commit/be661da7ea44969caaa62f041ffd5faed9764b14 linux-on-litex-vexriscv used `0x20000000` as `emulator_ram` base and `0xd0000000` as `spiflash` base.

Right now emulator is loaded into the main RAM under `0xc1100000`.